File: Data Structures & Algorithms/duplicate-integer/submission-1.cpp
class Solution {
public:
    bool hasDuplicate(vector<int>& nums) {
        sort(nums.begin(), nums.end());
        int end = nums.size() - 1;
        for (int i = 0; i < end; i++){
            if (nums.at(i) == nums.at(i+1))
                return true;
        }
        return false;
    }
};

File: Data Structures & Algorithms/max-consecutive-ones/submission-1.py
class Solution:
    def findMaxConsecutiveOnes(self, nums: List[int]) -> int:
        count = maxCount = 0
        for num in nums:
            count = count + 1 if num == 1 else 0
            if maxCount < count:
                maxCount = count
        return maxCount

